Fix TPS UI elements broken by XML -> JSON migration

ckelleyRH opened the pull request:
The field names were erroneously changed by not adding JSON tags to
rename the fields like the original XML implementation.

fmarco76 reviewed (approved):
LGTM.
There are a couple of tests failing but the error does not seem related to this PR

ckelleyRH commented:
> LGTM. There are a couple of tests failing but the error does not seem related to this PR

I'm not sure, it is complaining for example that it can't find configuration properties in one test, and I changed the serialisation of the `ConfigData` properties. I think too many tests failed here for it to not be an issue I introduced.

ckelleyRH commented:
It appears new code depending on the erroneous names has been introduced, hopefully it is just this one place in the deployment script!

ckelleyRH commented:
Thanks @edewata @fmarco76 !
